internal/experiments/experiments.go

```diff
@@ -1,32 +1,48 @@
 package experiments
 
 import (
-	"context"
-
-	"github.com/simskij/xk6-chaos/internal/experiments/podkillers"
-	"go.k6.io/k6/js/common"
+	"github.com/dop251/goja"
+	"github.com/grafana/xk6-chaos/internal/experiments/podkillers"
 	"go.k6.io/k6/js/modules"
 )
 
-
 // Register the extension on module initialization, available to
 // import from JS as "k6/x/chaos/experiments".
 func init() {
-	modules.Register("k6/x/chaos/experiments", &Experiments{
+	modules.Register("k6/x/chaos/experiments", &ExperimentsRoot{
 		// Podkiller: podkillers.New(),
 	})
 	// i++
 }
 
+var _ modules.Module = ExperimentsRoot{}
+
+type ExperimentsRoot struct{}
+
+func (k ExperimentsRoot) NewModuleInstance(vu modules.VU) modules.Instance {
+	return &Experiments{
+		vu: vu,
+	}
+}
+
 // This exposes experiment metadata for use in displaying results.
 type Experiments struct {
 	Podkiller *podkillers.Podkillers
 	Summary   string
+	vu        modules.VU
+}
+
+func (e *Experiments) Exports() modules.Exports {
+	return modules.Exports{
+		Named: map[string]interface{}{
+			"Podkillers": e.XPodkillers,
+		},
+	}
 }
 
 // XPodkillers serves as a constructor of the Podkillers js class
-func (*Experiments) XPodkillers(ctx *context.Context) (interface{}, error) {
-	rt := common.GetRuntime(*ctx)
+func (e *Experiments) XPodkillers(goja.ConstructorCall) goja.Object {
```

Comment:
The return type should be `*goja.Object` otherwise getting error running the script.

Reply:
:facepalm: I likely wrote it wrong from memory and then neovim's autocomplete fixed the type in the return :facepalm:

Thanks :bow:  it's fixed in  [e01e7ca](https://github.com/grafana/xk6-chaos/pull/5/commits/e01e7caf5b7c8b94886a3f09cbb1c8a9687d824d)